app/assets/stylesheets/components/_form.sass

```diff
@@ -0,0 +1,13 @@
+$Form-spacing-small: $Theme-spacing-small
+$Form-spacing-base: $Theme-spacing-base
+
+.Form
+
+.Form-item
+  padding: $Form-spacing-small
+
+.Form-field
+  padding: 10px
```

Comment:
This value isn't suposed to be here. I'm going to change it to `Form-spacing-small`